test/e2e/features/cat-mod2/01-pass-journey.feature

```diff
@@ -0,0 +1,70 @@
+@catm2 @full_smoke @regression
+
+  Feature: Driver Examiner complete the pass journey for Mod2
+
+    Background:
+      Given I am logged in as "desexamineram2" and I have a test for "Dr Fox Farrell"
+      When I start the test for "Dr Fox Farrell"
+      And the candidate completes the declaration page
+      And the candidate confirms their declaration
+      Then I should see the "Declaration - Fox Farrell" page
+      When the candidate requests to receive results by post
+      And I proceed to the bike
+
+      Scenario: Examiner changes test category to A
```

Comment:
Change this scenario title to something that makes sense. Like 'Driving Examiner Passes mod2 a candidate'

Reply:
done